lib/pageflow/configuration.rb

```diff
@@ -465,6 +465,13 @@ def enable_all_features
       features.enable_all(FeatureLevelConfiguration.new(self))
     end
 
+    # @api private
+    def share_providers_input_collection
+      available_share_providers.map do |provider|
+        [provider.to_s.camelize, provider]
+      end
+    end
+
```

Comment:
Let's move this to an admin helper. There are already other helpers that provide collections like this (e.g. [`Pageflow::Adimin::WidgetsHelper#admin_widget_types_collection_for_role`](https://github.com/codevise/pageflow/blob/a56c86ba90b987fe3a83a495b658540ba52ac433/app/helpers/pageflow/admin/widgets_helper.rb#L11)).